app_data/pages.yaml

```diff
@@ -0,0 +1,11 @@
+- url: /communities/horizen/info # The pid_value i.e. (horizen) to be replaced according to the respective branded community
```

Comment:
question: where does the `horizen` value came from?

Reply:
it is the pid_value of the themed community on my local